Formula/termius.rb

```diff
@@ -0,0 +1,175 @@
+class Termius < Formula
+  desc "CLI tool for termius.com (aka serverauditor.com)"
+  homepage "https://serverauditor.com"
+
+  stable do
```

Comment:
Don't need to use `stable do` when there's just the URL and SHA256 in the block.